gpu-service/jobs.py

```
@@ -0,0 +1,335 @@
+from retinaface import RetinaFace
+from moviepy import VideoFileClip
+from speechbrain.inference import EncoderDecoderASR
+from qdrant_client import QdrantClient, models
+
+# Add compatibility fixes for TensorFlow/Keras
+import warnings
+warnings.filterwarnings('ignore', category=UserWarning)
+warnings.filterwarnings('ignore', category=DeprecationWarning)
+
+# Fix for TensorFlow/Keras compatibility
+import tensorflow as tf
```

Ruff (0.11.9)

Module level import not at top of file

(E402)